tutorials/gui_for_python_applications/README.md

```diff
@@ -0,0 +1,187 @@
+# Adding a GUI to Holoscan Python Applications
+
+When developing Holoscan applications, incorporating a graphical user interface (GUI) can enhance usability and allow modification of the application's behavior at runtime.
+
+This tutorial demonstrates how GUI controls were integrated into the [Florence-2](https://github.com/nvidia-holoscan/holohub/tree/main/applications/florence-2-vision) Python application using [PySide6](https://doc.qt.io/qtforpython-6/). This addition enables users to dynamically change the vision task performed by the application.
+
+<p align="center">
+  <img src="./demo.gif" alt="Holoscan VILA Live">
+</p>
+
+## Overview
+
+The Florence-2 application includes the typical components of a Holohub application, with the addition of a GUI component. 
+The main components are:
+
+- <b>Dockerfile</b>: For installing additional dependencies.
+- <b>Application Code</b>: Defines the Holoscan application and its operators.
+- <b>GUI Code</b>: Utilizes PySide6 to add UI controls.
+
+
+### Dockerfile
+
+The Dockerfile is used in Holohub when the application requires additional dependencies. For GUI functionality, this application's 
+[Dockerfile](https://github.com/nvidia-holoscan/holohub/blob/main/applications/florence-2-vision/Dockerfile) installs:
+
+* `qt6-base-dev` for Qt6 framework
+* `PySide6` for Python bindings for Qt6 (as specified in [requirements.txt](https://github.com/nvidia-holoscan/holohub/blob/main/applications/florence-2-vision/requirements.txt))
+
+### Application Code
+
+The Florence-2 application code is organized across several files:
+
+* `florence2_app.py`: Main application code.
+* `florence2_op.py`: Florence-2 model inference code.
+* `florence2_postprocessor_op.py`: Post-processing code to send overlays (e.g., bounding boxes, labels, segmentation masks) to Holoviz.
+* `config.yaml`: Default application parameters.
+
+The Florence-2 application operates independently of the GUI code. You can run it with `python application/florence-2-vision/florence2_app.py` inside the Florence-2 Docker container. For GUI integration, the `set_parameters()` method was added to the `FlorenceApp` class. This method updates two fields in the Florence-2 operator:
```

Comment:
>The Florence-2 application operates independently of the GUI code

What does this mean in context?
- GUI controls can be disabled?
- The Florence-2 pipeline runs as a separate process from GUI controls?
- Does running `florence2_app.py` directly exclude GUI controls?

Reply:
It means we can run florence-2 without gui controls, more to emphasize that the application logic and the gui logic is not tightly coupled.  so user can develop their app code first and then easily add gui to their app by adding the qt_app.py (with some modifications to qt_app.py)

Reply:
updated the wording here a little